python/tvm/target/target.py

```diff
@@ -642,6 +642,27 @@ def create_tvm_options(cpu_ver, config):  # pylint: disable=unused-argument
 
     return Target(" ".join(["hexagon"] + args_list))
 
+STM32_SUPPORTED_MODELS = {
+    #"stm32H7xx": ["-device=arm_cpu", "-mcpu=cortex-m7", "-march=armv7e-m"],
+    "stm32H7xx": ["-device=arm_cpu", "-mcpu=cortex-m7"],
```

Comment:
> We first aligned with the CubeAI libraries that have a library per series.
Do these libraries contain implemented kernels? (e.g. do you intend to integrate them similarly to CMSIS-NN?)

if we move to one board id per target string, won't there be quite a lot of repetition? possible to compress the table somehow (e.g. just look at the board id prefix)? at least, perhaps we could start with something like that? in general we'd like to avoid manual per-board configuration in TVM if possible (but I'm asking the question above in case we need to consider that).

Reply:
Hello, Andrew
No, we do not intend to integrate the CubeAI libraries with the TVM.
Let me think how to compact the target options table.

Reply:
Back to specifying the series.
There are 17 series altogether - I have listed 14, did not list connectivity series.
Although multiple series refer to the same CPU, the differences exist with their memory configurations. This will show in the schedules that TVM will have to generate. I am working on different schedules for different targets.
Arthur

Reply:
@areusch 
Have forgotten this ...